### PR TITLE
Make the details of what the Okta CLI does collapsible

### PR DESCRIPTION
**Pull request by mraible**
This PR makes it so the Okta CLI include doesn't take up so much space on the page. After this change, the details are in a collapsible block. The screenshots below are from our [Angular + Bootstrap post](https://developer.okta.com/blog/2020/03/02/angular-bootstrap).

<details>
<summary>Before the CLI include, setup instructions fit on one page.</summary>
bootstrap-before
</details>

<details>
<summary>In its current state, the Okta CLI takes up a lot more space.</summary>
bootstrap-after
</details>

<details>
<summary>With this change, everything fits on one page again.</summary>
bootstrap-details
</details>

The only downside is the "details" have to be in Markdown. AsciiDoc no longer works within it.

### CODE REVIEW

**Comment by bdemers** [comment]
Nice!!

**Review by bdemers — APPROVED** [review]

**Comment by mraible** [comment]
I still need to test this in a few more browsers.

**Comment by mraible** [comment]
Fixed it in IE/Edge with good ol' Stack Overflow: https://stackoverflow.com/a/53113879/65681

File: _source/_includes/footer.html [hunk 1]
```diff
@@ -84,3 +84,5 @@ <h4 class="column-header">More Info</h4>
     {% if layout.js %}
       {% js '{{layout.js}}' %}
     {% endif %}
+    <!-- Fix <details> for IE/Edge -->
+    <script src="//cdn.jsdelivr.net/npm/details-polyfill@1/index.min.js" async></script>
```
Comment (mraible) [inline _source/_includes/footer.html:88]: Is this the right place to put new scripts @rdegges @hamzaessaoui-okta?

Comment (mraible) [reply _source/_includes/footer.html:88]: No response means yes! :)

Comment (hamzaessaoui-okta) [reply _source/_includes/footer.html:88]: Sorry @mraible , just seeing this :)

but yes, that's a good place to load js

**Pull request merged**